src/components/concept/_lang-ssc-features.md

```diff
@@ -0,0 +1,18 @@
+<dl>
+<dt>Reachability analysis</dt>
+<dd>
+Reachability refers to whether or not a vulnerable code pattern from a dependency is used in the codebase that imports it. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match for a vulnerability to be considered reachable.
+</dd>
+<dt>License detection</dt>
+<dd>
+Semgrep Supply Chain's **license compliance** feature enables you to explicitly allow or disallow (block) a package's use in your repository based on its license. For example, your company policy may disallow the use of packages with the Creative Commons Attribution-NonCommercial (CC-BY-NC) license.
```

Comment:
```suggestion
Semgrep Supply Chain's **license compliance** feature enables you to explicitly allow or disallow (block) a package's use in your repository based on its license. For example, your company policy may disallow the use of packages with the Creative Commons Attribution-NonCommercial (CC-BY-NC) license. Semgrep can help enforce this restriction.
```
I'm not attached to the wording, but something to indicate what Semgrep can do if someone tries to use a disallowed package would be helpful